### PR TITLE
TM-5233-Employee-Agenda-Suffix

### PR DESCRIPTION
**Pull request by cagbakwu**
[ticket](https://metaphase.atlassian.net/browse/TM-5233)
Screenshot 2024-03-04 at 1 47 40 PM
Screenshot 2024-03-04 at 1 44 37 PM

### CODE REVIEW

File: talentmap_api/fsbid/services/agenda_employees.py [hunk 1]
```diff
@@ -223,7 +221,7 @@ def fsbid_agenda_employee_to_talentmap_agenda_employee(data, cdos=[]):
         cdo = cdoObj
     return {
         "person": {
-            "fullName": fullName + f"{', '+suffix if hasSuffix else ''} ",
+            "fullName": fullName,
```
Comment (cagbakwu) [inline talentmap_api/fsbid/services/agenda_employees.py:224]: The 'fullname' already contains the suffix so adding it to the end would be unnecessary.

**Comment by cagbakwu** [comment]
Will remove WIP after conversation with Tim.

File: talentmap_api/fsbid/services/agenda_employees.py (hunk 1 above)
Comment (elizabeth-jimenez) [reply talentmap_api/fsbid/services/agenda_employees.py:224]: sometimes it doesn't tho

**Comment by elizabeth-jimenez** [comment]
Changes Requested
- dig deeper into EP to see how the data is returned

File: talentmap_api/fsbid/services/agenda_employees.py (hunk 1 above)
Comment (elizabeth-jimenez) [reply talentmap_api/fsbid/services/agenda_employees.py:224]: on further research, `fullname` never seems to include the suffix

File: talentmap_api/fsbid/services/agenda_employees.py [hunk 2]
```diff
@@ -210,8 +210,8 @@ def fsbid_agenda_employee_to_talentmap_agenda_employee(data, cdos=[]):
     lastN = data.get('perpiilastname', '')
     initials = f"{firstN[0] if firstN else ''}{lastN[0] if lastN else ''}"
     suffix = data.get('perpiisuffixname') or ''
-    hasSuffix = len(suffix.strip()) > 0
     fullName = data.get("perpiifullname", "")
+    hasSuffix = len(suffix.strip()) > 0 and 'JR-' not in fullName
```
Comment (elizabeth-jimenez) [inline talentmap_api/fsbid/services/agenda_employees.py:214]: 'JR-' is not a suffix - that is scrambled data and you have to dig deeper to see that it is just part of the last or first name and not a true suffix

**Pull request closed without merging**